src/components/AdditionalInformation.jsx

```diff
@@ -279,13 +279,18 @@ const AdditionalInformation = (props) => {
             </div>
           </div>
         )}
-        <div className="d-flex justify-content-between">
-          <SeButton text="Previous" onClick={callPreviousForm} />
+        <div className="d-md-flex justify-content-md-between d-sm-block">
+          <SeButton
+            text="Previous"
+            onClick={callPreviousForm}
+            className="d-sm-block mb-3"
+          />
           <SeButton
             text="File Your Report"
             onClick={SubmitTheForm}
             isLoading={formik.isSubmitting}
             isLoadingText="Submitting Request..."
+            className="d-sm-block mb-3"
```

Comment:
should all SeButtons have "d-sm-block mb-3" ?

Reply:
@martypowell so maybe not but when I only add it to the top button when the screen is full size the button without the margin expands to in height.

![image](https://user-images.githubusercontent.com/40430541/83521969-fd0ce680-a4ad-11ea-9ffb-223c87a9f36e.png)

Reply:
So maybe there is another solution like just adding a style to the first button of padding-bottom: 3px which wont make the other buttons height change I havent tried that yet

Reply:
something doesn't seem right here, let me take a quick look.